crux-test/test/crux/current_queries_test.clj

```diff
@@ -22,27 +25,68 @@
                   ::query-id 3}]]
     (t/testing "test recent-queries - check max count expiration"
       (t/is (= [1]
-               (->> (clean-expired-queries queries
-                                           {::n/recent-queries-max-age (Duration/ofSeconds 8)
-                                            ::n/recent-queries-max-count 1})
+               (->> (clean-completed-queries queries
+                                             {::n/recent-queries-max-age (Duration/ofSeconds 8)
+                                              ::n/recent-queries-max-count 1})
                     (map ::query-id))))
 
       (t/is (= [1 2]
-               (->> (clean-expired-queries queries
-                                           {::n/recent-queries-max-age (Duration/ofSeconds 8)
-                                            ::n/recent-queries-max-count 2})
+               (->> (clean-completed-queries queries
+                                             {::n/recent-queries-max-age (Duration/ofSeconds 8)
+                                              ::n/recent-queries-max-count 2})
                     (map ::query-id)))))
 
     (t/testing "test recent-queries - check time expiration"
       (t/is (= [1]
-               (->> (clean-expired-queries queries
-                                           {::n/recent-queries-max-age (Duration/ofSeconds 4)
-                                            ::n/recent-queries-max-count 5})
+               (->> (clean-completed-queries queries
+                                             {::n/recent-queries-max-age (Duration/ofSeconds 4)
+                                              ::n/recent-queries-max-count 5})
                     (map ::query-id))))
       (t/is (= [1 2]
-               (->> (clean-expired-queries queries
-                                           {::n/recent-queries-max-age (Duration/ofSeconds 8)
-                                            ::n/recent-queries-max-count 5})
+               (->> (clean-completed-queries queries
+                                             {::n/recent-queries-max-age (Duration/ofSeconds 8)
+                                              ::n/recent-queries-max-count 5})
+                    (map ::query-id))))))
+  (let [clean-slowest-queries @#'n/clean-slowest-queries
```

Comment:
given that these are now two code paths I'd split the tests into a different `t/deftest` - means that if one side breaks in the future you rerun fewer tests while you're fixing it